mozilla/blurts-server
# Fix “Learn more link” on the “Start free scan” page

# References: 
Jira: [MNTOR-2263](https://mozilla-hub.atlassian.net/browse/MNTOR-2263)



# Description

Adds the correct link to the “How it works” page.

# How to test

1. Visit http://localhost:6060/redesign/user/dashboard/fix/data-broker-profiles/start-free-scan
2. Clicking the “Learn more” link should open the correct page in a new tab or window

# Checklist (Definition of Done)


[MNTOR-2263]: https://mozilla-hub.atlassian.net/browse/MNTOR-2263?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
